todoman/model.py

```diff
@@ -236,30 +237,52 @@ class Database:
 
     def __init__(self, path):
         self.path = path
+        self._cache_path = os.path.join(self.path, '.todoman-cache')
```

Comment:
$XDG_DATA_HOME sounds okay for this:

https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#variables